### ISSUE TITLE
Formatter adds additional lines between package and option

### ISSUE THREAD

**Issue by zzwx** [devkanro/intellij-protobuf-plugin#14]
Example: 
```
syntax = "proto3";
package test;

option go_package = "test/grpc/pb";
```

Hit Ctrl-Alt-L, watch 2 empty lines added above `option`.

**Comment by zzwx** [comment]
Additional info. That doesn't happen if an `import` statement is between `package` and `option`.

**Comment by zzwx** [comment]
A guess: an artifact from auto-removal portion of unused imports.

**Comment by devkanro** [comment]
![bug](https://user-images.githubusercontent.com/9367842/118160616-1bff5100-b451-11eb-942a-b74e22b38619.gif)

I can't reproduce it. can you provide your proto before formatted?

**Comment by devkanro** [comment]
Reproduce it after check `import optimizer` option in formatter.

**Comment by devkanro** [comment]
Fixed in cf35cf21efd0166b82e6e7471f0335be31a13497

Additionally, I add line break settings in code style setting page, just change it if you have your own style.
![Snipaste_2021-05-14_02-33-36](https://user-images.githubusercontent.com/9367842/118170679-f8da9e80-b45c-11eb-981b-610c8c3e964b.png)

**Comment by zzwx** [comment]
Amazing, you are so fast!

**Comment by devkanro** [comment]
I planned to release it with symbol resolver in quick document, it dependent on commonmark/commonmark-java#205 merged

**Comment by devkanro** [comment]
Before released you can test it with 1.3.0-SNAOSHOT
[IntelliJ.Protobuf.Language.Plugin-1.3.0.zip](https://github.com/devkanro/intellij-protobuf-plugin/files/6477168/IntelliJ.Protobuf.Language.Plugin-1.3.0.zip)

**Comment by zzwx** [comment]
Well done! Thank you!
